server/chat-commands/info.ts

```diff
@@ -491,6 +491,8 @@ export const commands: ChatCommands = {
 				Utils.html`${curUser.connected ? ` \u25C9 ` : ` \u25CC `} ${curUser.name}`
 			);
 		}
+		Utils.sortBy(results, name => name.startsWith('  \u25C9'));
+		if (!results.length) return this.errorReply(`No users found.`);
```

Comment:
```suggestion
		if (!results.length) results = [`No users found.`];
```

Gives a nicer error message which reminds you what you searched for.